uber/templates/regform.html

```diff
@@ -624,19 +621,23 @@
         });
     {% endif %}
 
-    {% if is_prereg_confirm_email_enabled %}
-      $(function () {
-        $('form[action="post_form"]').validate({
+    $(function () {
+      $('form').validate({
+        {% if is_prereg_confirm_email_enabled %}
```

Comment:
Now we're ALWAYS using the jQuery Validation framework to validate the form. Works almost the same as regular HTML5 validation. If `prereg_confirm_email_enabled` is not `true` then we skip trying to validate the email confirmation field.